components/SearchBar/SearchBar.tsx

```diff
@@ -145,10 +145,14 @@ const SearchBar = ({ big }: SearchProps) => {
     search(word)
   }
 
-  function handleOnChange(el) {
-    setSearchValue(el)
-    setSearchSuggestedWords(true)
-    searchValue !== ('' || undefined) && setIsSuggestionOpen(true)
+  function handleOnChange(value: string) {
+    setSearchValue(value)
+
+    const trimmedValue = value.trim()
+    if (trimmedValue) {
```

Comment:
Why create a new variable if you are not using it anywhere else. I say pass in `value.trim()` as the parameter to the `if` statement.

Reply:
> 10 lines of code = 10 issues.
> 500 lines of code = “looks fine.”
> Code reviews.
>  — [@iamdevloper](https://twitter.com/iamdevloper/status/397664295875805184)

Heard of that? :)

But you're right. Will change accordingly.